pom.xml

```diff
@@ -73,6 +73,21 @@
 			<version>2.10.1</version>
 		</dependency>
 
+		<!-- UNIX socket connection support -->
+		<dependency>
+			<groupId>com.kohlschutter.junixsocket</groupId>
+			<artifactId>junixsocket-core</artifactId>
```

Comment:
If we're introducing this, why can we use the latest, 2.8.1?

Reply:
@chayim We're not introducing this. I have just moved it up to keep "the test dependencies that can be used as optional dependencies" together.

Reply:
Diffs clearly aren't showing right here :(.  Okay, so same question - if it's a test dependency can we upgrade this to latest while we're here?

Reply:
Done.